JacobEkedahl/detect-intros-from-video
# Svtplay wrapper

The task #28 

## To test
- cd detect-intros-from-video
- pip install -r requirements.txt
- - python main.py --downloadvideos video-serier.txt svtplay-dir 2
- make sure you have the video-serier.txt in home directory
